src/icon/index.tsx

```diff
@@ -1,111 +1,160 @@
+import AirConditioningIcon from './airConditioning'
```

Comment:
Don't you want to add the `Icon` suffix for all to be consistent with the existing ones?

Reply:
It' done on purpose some existing ones are missing the icon suffix, so I didn't introduce a new way, it was already hybrid. Also we don't need this in the filename as long as they are exporting as `AirConditioningIcon`.
I'd rather remove icon suffix from all files in their filename. WDYT?

Reply:
I'm OK as long as we're consistent. So: suffix in exported components, no suffix in directories?

Reply:
Yep, but that would mean breaking the paths already used in the lib and the SPA. Let's think about it.

Reply:
I'd say we choose to remove them as a rule, but the change should take place in another PR